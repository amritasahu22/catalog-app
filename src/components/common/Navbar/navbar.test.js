import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import NavBar from "./navbar";

test("should render properly", () => {
	const tree = shallow(<NavBar />);
	expect(toJson(tree)).toMatchSnapshot();
});
