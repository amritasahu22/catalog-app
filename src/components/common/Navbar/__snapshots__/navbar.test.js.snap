// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<nav
  className="navbar navbar-expand-md navbar-dark bg-dark fixed-top"
>
  <a
    className="navbar-brand"
  >
    THE ICONIC
  </a>
  <button
    aria-controls="navbarsExampleDefault"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarsExampleDefault"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    id="navbarsExampleDefault"
  >
    <ul
      className="navbar-nav mr-auto"
    >
      <li
        className="nav-item active"
      >
        <a
          className="nav-link"
        >
          Home 
          <span
            className="sr-only"
          >
            (current)
          </span>
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
        >
          Women
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
